app/views/data_access/configure.html.erb

```diff
@@ -39,6 +39,7 @@
             <p class="summary" data-bind="if: granuleAccessOptions().size && granuleAccessOptions().size != 0">
               <span class="summary-data" data-bind="text: granuleAccessOptions().size"></span>
               <span class="summary-label" data-bind="text: granuleAccessOptions().sizeUnit"></span>
+              <span class="summary-caveat" data-bind="if: collection.cwic()">This is <span class="cwic-label" data-bind="text: collection.granuleDescription"></span> data. Granule size is not available. </span>
```

Comment:
As mentioned in the overall comment, I think this should be removed. Just make sure `granuleAccessOptions().size` is null or 0 and the whole thing won't be displayed, which is what we do for CMR.